app/xl_NewFileByTemplate.py

```diff
@@ -73,7 +73,7 @@ def SetUp(xlApp):
 			ws = xlApp.Sheets(tempSheetName)
 			ws.Activate
 	        
-			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName[i]) + ".xlsx")
+			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName) + ".xlsx")
 			xlApp.ActiveWorkbook.Close(False)
 			xlApp.screenUpdating = True
 			Marshal.ReleaseComObject(ws)
```

Comment:
Will do! This is my first attempt at a PR ever so the opportunity and feedback are appreciated. :)